src/main.rs

```diff
@@ -85,6 +85,22 @@ async fn main() -> miette::Result<()> {
                         get_build_output(&build_args, recipe_path.clone(), &tool_config).await?;
                     outputs.extend(output);
                 }
+
+                if build_args.render_only {
+                    let mut render_output = Vec::new();
+                    outputs.iter().for_each(|output| {
+                        let resolved_dep_as_str: Vec<String> = output
+                            .finalized_dependencies
+                            .iter()
+                            .map(|resolved_dep| serde_json::to_string_pretty(resolved_dep).unwrap())
+                            .collect();
+                        render_output.extend(resolved_dep_as_str);
+                    });
+
+                    println!("[{}]", render_output.join(","));
+                    return Ok(());
+                }
```

Comment:
I think this can be simplified as:

```suggestion
                if build_args.render_only {
                    let render_output: Vec<FinalizedDependencies> = outputs
                        .into_iter()
                        .flat_map(|output| output.finalized_dependencies)
                        .collect();
                    println!(
                        "{}",
                        serde_json::to_string_pretty(&render_output).into_diagnostic()?
                    );
                    return Ok(());
                }

```

What do you think? 🐻

Reply:
In this case I can't parse this json output, because it will just some without "," comma delimiter and not array - so it will error if I will try to load this

Reply:
it will be like this:
```
{
  "a": "b"
}

{
  "a": "b"
}

{
  "a": "b"
}
```

but I need this:
```
[{
  "a": "b"
}, 
{
  "a": "b"
}]
```

Reply:
I'm getting a JSON array with `rattler-build build -r examples/linking/ -r test-data/recipes/variant_config/ --render-only`

<details>
<summary>Details</summary>

```json
[
  {
    "build": {
      "specs": [
        {
          "source": "compiler",
          "spec": "gcc_linux-64"
        },
        {
          "source": "raw",
          "spec": "ninja"
        },
        {
          "source": "raw",
          "spec": "cmake"
        }
      ],
      "resolved": [
        {
          "build": "h924138e_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=12",
            "libstdcxx-ng >=12"
          ],
          "license": "Apache-2.0",
          "license_family": "Apache",
          "md5": "73a4953a2d9c115bdc10ff30a52f675f",
          "name": "ninja",
          "sha256": "b555247ac8859b4ff311e3d708a0640f1bfe9fae7125c485b444072474a84c41",
          "size": 2251263,
          "subdir": "linux-64",
          "timestamp": 1676837602636,
          "version": "1.11.1",
          "fn": "ninja-1.11.1-h924138e_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/ninja-1.11.1-h924138e_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h807b86a_5",
          "build_number": 5,
          "constrains": [
            "libgomp 13.2.0 h807b86a_5"
          ],
          "depends": [
            "_libgcc_mutex 0.1 conda_forge",
            "_openmp_mutex >=4.5"
          ],
          "license": "GPL-3.0-only WITH GCC-exception-3.1",
          "license_family": "GPL",
          "md5": "d4ff227c46917d3b4565302a2bbb276b",
          "name": "libgcc-ng",
          "sha256": "d32f78bfaac282cfe5205f46d558704ad737b8dbf71f9227788a5ca80facaba4",
          "size": 770506,
          "subdir": "linux-64",
          "timestamp": 1706819192021,
          "version": "13.2.0",
          "fn": "libgcc-ng-13.2.0-h807b86a_5.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libgcc-ng-13.2.0-h807b86a_5.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "conda_forge",
          "build_number": 0,
          "depends": [],
          "license": "None",
          "md5": "d7c89558ba9fa0495403155b64376d81",
          "name": "_libgcc_mutex",
          "sha256": "fe51de6107f9edc7aa4f786a70f4a883943bc9d39b3bb7307c04c41410990726",
          "size": 2562,
          "subdir": "linux-64",
          "timestamp": 1578324546067,
          "version": "0.1",
          "fn": "_libgcc_mutex-0.1-conda_forge.tar.bz2",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/_libgcc_mutex-0.1-conda_forge.tar.bz2",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "2_gnu",
          "build_number": 16,
          "constrains": [
            "openmp_impl 9999"
          ],
          "depends": [
            "_libgcc_mutex 0.1 conda_forge",
            "libgomp >=7.5.0"
          ],
          "license": "BSD-3-Clause",
          "license_family": "BSD",
          "md5": "73aaf86a425cc6e73fcf236a5a46396d",
          "name": "_openmp_mutex",
          "sha256": "fbe2c5e56a653bebb982eda4876a9178aedfc2b545f25d0ce9c4c0b508253d22",
          "size": 23621,
          "subdir": "linux-64",
          "timestamp": 1650670423406,
          "version": "4.5",
          "fn": "_openmp_mutex-4.5-2_gnu.tar.bz2",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/_openmp_mutex-4.5-2_gnu.tar.bz2",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h807b86a_5",
          "build_number": 5,
          "depends": [
            "_libgcc_mutex 0.1 conda_forge"
          ],
          "license": "GPL-3.0-only WITH GCC-exception-3.1",
          "license_family": "GPL",
          "md5": "d211c42b9ce49aee3734fdc828731689",
          "name": "libgomp",
          "sha256": "0d3d4b1b0134283ea02d58e8eb5accf3655464cf7159abf098cc694002f8d34e",
          "size": 419751,
          "subdir": "linux-64",
          "timestamp": 1706819107383,
          "version": "13.2.0",
          "fn": "libgomp-13.2.0-h807b86a_5.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libgomp-13.2.0-h807b86a_5.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h7e041cc_5",
          "build_number": 5,
          "depends": [],
          "license": "GPL-3.0-only WITH GCC-exception-3.1",
          "license_family": "GPL",
          "md5": "f6f6600d18a4047b54f803cf708b868a",
          "name": "libstdcxx-ng",
          "sha256": "a56c5b11f1e73a86e120e6141a42d9e935a99a2098491ac9e15347a1476ce777",
          "size": 3834139,
          "subdir": "linux-64",
          "timestamp": 1706819252496,
          "version": "13.2.0",
          "fn": "libstdcxx-ng-13.2.0-h7e041cc_5.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libstdcxx-ng-13.2.0-h7e041cc_5.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h1ed452b_3",
          "build_number": 3,
          "depends": [
            "binutils_linux-64 2.40 hdade7a5_3",
            "gcc_impl_linux-64 13.2.0.*",
            "sysroot_linux-64"
          ],
          "license": "BSD-3-Clause",
          "license_family": "BSD",
          "md5": "0aba5ba9908f419076c59e99ad30e6ee",
          "name": "gcc_linux-64",
          "sha256": "00e99542a7a86ed4d79f19b5f40b99870898663cd1ff0cfa27f83a0b4ff8c7e8",
          "size": 30745,
          "subdir": "linux-64",
          "timestamp": 1710260218568,
          "version": "13.2.0",
          "fn": "gcc_linux-64-13.2.0-h1ed452b_3.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/gcc_linux-64-13.2.0-h1ed452b_3.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hdade7a5_3",
          "build_number": 3,
          "depends": [
            "binutils_impl_linux-64 2.40.*",
            "sysroot_linux-64"
          ],
          "license": "BSD-3-Clause",
          "license_family": "BSD",
          "md5": "2d9a60578bc28469d9aeef9aea5520c3",
          "name": "binutils_linux-64",
          "sha256": "d114b825acef51c1d065ca0a17f97e0e856c48765aecf2f8f164935635013dd2",
          "size": 28868,
          "subdir": "linux-64",
          "timestamp": 1710259805994,
          "version": "2.40",
          "fn": "binutils_linux-64-2.40-hdade7a5_3.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/binutils_linux-64-2.40-hdade7a5_3.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hf600244_0",
          "build_number": 0,
          "depends": [
            "ld_impl_linux-64 2.40 h41732ed_0",
            "sysroot_linux-64"
          ],
          "license": "GPL-3.0-only",
          "license_family": "GPL",
          "md5": "33084421a8c0af6aef1b439707f7662a",
          "name": "binutils_impl_linux-64",
          "sha256": "a7e0ea2b71a5b03d82e5a58fb6b612ab1c44d72ce161f9aa441f7ba467cd4c8d",
          "size": 5414922,
          "subdir": "linux-64",
          "timestamp": 1674833958334,
          "version": "2.40",
          "fn": "binutils_impl_linux-64-2.40-hf600244_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/binutils_impl_linux-64-2.40-hf600244_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h41732ed_0",
          "build_number": 0,
          "constrains": [
            "binutils_impl_linux-64 2.40"
          ],
          "depends": [],
          "license": "GPL-3.0-only",
          "license_family": "GPL",
          "md5": "7aca3059a1729aa76c597603f10b0dd3",
          "name": "ld_impl_linux-64",
          "sha256": "f6cc89d887555912d6c61b295d398cff9ec982a3417d38025c45d5dd9b9e79cd",
          "size": 704696,
          "subdir": "linux-64",
          "timestamp": 1674833944779,
          "version": "2.40",
          "fn": "ld_impl_linux-64-2.40-h41732ed_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/ld_impl_linux-64-2.40-h41732ed_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h338b0a0_5",
          "build_number": 5,
          "depends": [
            "binutils_impl_linux-64 >=2.39",
            "libgcc-devel_linux-64 13.2.0 ha9c7c90_105",
            "libgcc-ng >=13.2.0",
            "libgomp >=13.2.0",
            "libsanitizer 13.2.0 h7e041cc_5",
            "libstdcxx-ng >=13.2.0",
            "sysroot_linux-64"
          ],
          "license": "GPL-3.0-only WITH GCC-exception-3.1",
          "license_family": "GPL",
          "md5": "a6be13181cb66a78544b1d5f7bac97d0",
          "name": "gcc_impl_linux-64",
          "sha256": "baab8f8b9af54959735e629cf6d5ec9378166aa4c68ba8dc98dc0a781d548409",
          "size": 53318565,
          "subdir": "linux-64",
          "timestamp": 1706819323755,
          "version": "13.2.0",
          "fn": "gcc_impl_linux-64-13.2.0-h338b0a0_5.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/gcc_impl_linux-64-13.2.0-h338b0a0_5.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h7e041cc_5",
          "build_number": 5,
          "depends": [
            "libgcc-ng >=13.2.0"
          ],
          "license": "GPL-3.0-only WITH GCC-exception-3.1",
          "license_family": "GPL",
          "md5": "3f686300a92604d1bdff9a29dd4a6639",
          "name": "libsanitizer",
          "sha256": "97ecdab7e4e96400d712c2d6ba2b7c30a97278e9f4470ea0ff36bf4f1447b3b9",
          "size": 4114208,
          "subdir": "linux-64",
          "timestamp": 1706819228913,
          "version": "13.2.0",
          "fn": "libsanitizer-13.2.0-h7e041cc_5.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libsanitizer-13.2.0-h7e041cc_5.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "ha9c7c90_105",
          "build_number": 105,
          "depends": [],
          "license": "GPL-3.0-only WITH GCC-exception-3.1",
          "license_family": "GPL",
          "md5": "3bc29a967fee57e193ce51f51c598bca",
          "name": "libgcc-devel_linux-64",
          "noarch": "generic",
          "sha256": "858029ad4d66869c533bb5a22e95e7c044ca66c61d6f403f10d9ae074a0e360e",
          "size": 2578210,
          "subdir": "noarch",
          "timestamp": 1706819085946,
          "version": "13.2.0",
          "fn": "libgcc-devel_linux-64-13.2.0-ha9c7c90_105.conda",
          "url": "https://conda.anaconda.org/conda-forge/noarch/libgcc-devel_linux-64-13.2.0-ha9c7c90_105.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "he073ed8_17",
          "build_number": 17,
          "depends": [
            "kernel-headers_linux-64 2.6.32 he073ed8_17"
          ],
          "license": "LGPL-2.0-or-later AND LGPL-2.0-or-later WITH exceptions AND GPL-2.0-or-later AND MPL-2.0",
          "license_family": "GPL",
          "md5": "595db67e32b276298ff3d94d07d47fbf",
          "name": "sysroot_linux-64",
          "noarch": "generic",
          "sha256": "b4e4d685e41cb36cfb16f0cb15d2c61f8f94f56fab38987a44eff95d8a673fb5",
          "size": 15127123,
          "subdir": "noarch",
          "timestamp": 1708000843849,
          "version": "2.12",
          "fn": "sysroot_linux-64-2.12-he073ed8_17.conda",
          "url": "https://conda.anaconda.org/conda-forge/noarch/sysroot_linux-64-2.12-he073ed8_17.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "he073ed8_17",
          "build_number": 17,
          "constrains": [
            "sysroot_linux-64 ==2.12"
          ],
          "depends": [],
          "license": "LGPL-2.0-or-later AND LGPL-2.0-or-later WITH exceptions AND GPL-2.0-or-later AND MPL-2.0",
          "license_family": "GPL",
          "md5": "d731b543793afc0433c4fd593e693fce",
          "name": "kernel-headers_linux-64",
          "noarch": "generic",
          "sha256": "fb39d64b48f3d9d1acc3df208911a41f25b6a00bd54935d5973b4739a9edd5b6",
          "size": 710627,
          "subdir": "noarch",
          "timestamp": 1708000830116,
          "version": "2.6.32",
          "fn": "kernel-headers_linux-64-2.6.32-he073ed8_17.conda",
          "url": "https://conda.anaconda.org/conda-forge/noarch/kernel-headers_linux-64-2.6.32-he073ed8_17.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hcfe8598_0",
          "build_number": 0,
          "depends": [
            "bzip2 >=1.0.8,<2.0a0",
            "libcurl >=8.6.0,<9.0a0",
            "libexpat >=2.6.2,<3.0a0",
            "libgcc-ng >=12",
            "libstdcxx-ng >=12",
            "libuv >=1.48.0,<2.0a0",
            "libzlib >=1.2.13,<1.3.0a0",
            "ncurses >=6.4.20240210,<7.0a0",
            "rhash >=1.4.4,<2.0a0",
            "xz >=5.2.6,<6.0a0",
            "zstd >=1.5.5,<1.6.0a0"
          ],
          "license": "BSD-3-Clause",
          "license_family": "BSD",
          "md5": "95b78e9d65446d1c5f2243c4e28e0f87",
          "name": "cmake",
          "sha256": "a49d73c2add8833069993a32caa4d04ca5763013cf893f7f3fb48ec26089e615",
          "size": 18983777,
          "subdir": "linux-64",
          "timestamp": 1711043354018,
          "version": "3.29.0",
          "fn": "cmake-3.29.0-hcfe8598_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/cmake-3.29.0-hcfe8598_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hfc55251_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=12",
            "libstdcxx-ng >=12",
            "libzlib >=1.2.13,<1.3.0a0"
          ],
          "license": "BSD-3-Clause",
          "license_family": "BSD",
          "md5": "04b88013080254850d6c01ed54810589",
          "name": "zstd",
          "sha256": "607cbeb1a533be98ba96cf5cdf0ddbb101c78019f1fda063261871dad6248609",
          "size": 545199,
          "subdir": "linux-64",
          "timestamp": 1693151163452,
          "version": "1.5.5",
          "fn": "zstd-1.5.5-hfc55251_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/zstd-1.5.5-hfc55251_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h166bdaf_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "LGPL-2.1 and GPL-2.0",
          "md5": "2161070d867d1b1204ea749c8eec4ef0",
          "name": "xz",
          "sha256": "03a6d28ded42af8a347345f82f3eebdd6807a08526d47899a42d62d319609162",
          "size": 418368,
          "subdir": "linux-64",
          "timestamp": 1660346797927,
          "version": "5.2.6",
          "fn": "xz-5.2.6-h166bdaf_0.tar.bz2",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/xz-5.2.6-h166bdaf_0.tar.bz2",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hd590300_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "MIT",
          "license_family": "MIT",
          "md5": "ec972a9a2925ac8d7a19eb9606561fff",
          "name": "rhash",
          "sha256": "12711d2d4a808a503c2e49b25d26ecb351435521e814c154e682dd2be71c2611",
          "size": 185144,
          "subdir": "linux-64",
          "timestamp": 1693455923632,
          "version": "1.4.4",
          "fn": "rhash-1.4.4-hd590300_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/rhash-1.4.4-hd590300_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h59595ed_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "X11 AND BSD-3-Clause",
          "md5": "97da8860a0da5413c7c98a3b3838a645",
          "name": "ncurses",
          "sha256": "aa0f005b6727aac6507317ed490f0904430584fa8ca722657e7f0fb94741de81",
          "size": 895669,
          "subdir": "linux-64",
          "timestamp": 1710866638986,
          "version": "6.4.20240210",
          "fn": "ncurses-6.4.20240210-h59595ed_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/ncurses-6.4.20240210-h59595ed_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hd590300_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "MIT",
          "license_family": "MIT",
          "md5": "7e8b914b1062dd4386e3de4d82a3ead6",
          "name": "libuv",
          "sha256": "b7c0e8a0c93c2621be7645b37123d4e8d27e8a974da26a3fba47a9c37711aa7f",
          "size": 899979,
          "subdir": "linux-64",
          "timestamp": 1709913354710,
          "version": "1.48.0",
          "fn": "libuv-1.48.0-hd590300_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libuv-1.48.0-hd590300_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h59595ed_0",
          "build_number": 0,
          "constrains": [
            "expat 2.6.2.*"
          ],
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "MIT",
          "license_family": "MIT",
          "md5": "e7ba12deb7020dd080c6c70e7b6f6a3d",
          "name": "libexpat",
          "sha256": "331bb7c7c05025343ebd79f86ae612b9e1e74d2687b8f3179faec234f986ce19",
          "size": 73730,
          "subdir": "linux-64",
          "timestamp": 1710362120304,
          "version": "2.6.2",
          "fn": "libexpat-2.6.2-h59595ed_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libexpat-2.6.2-h59595ed_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hca28451_0",
          "build_number": 0,
          "depends": [
            "krb5 >=1.21.2,<1.22.0a0",
            "libgcc-ng >=12",
            "libnghttp2 >=1.58.0,<2.0a0",
            "libssh2 >=1.11.0,<2.0a0",
            "libzlib >=1.2.13,<1.3.0a0",
            "openssl >=3.2.1,<4.0a0",
            "zstd >=1.5.5,<1.6.0a0"
          ],
          "license": "curl",
          "license_family": "MIT",
          "md5": "704739398d858872cb91610f49f0ef29",
          "name": "libcurl",
          "sha256": "357ce806adf1818dc8dccdcd64627758e1858eb0d8a9c91aae4a0eeee2a44608",
          "size": 391187,
          "subdir": "linux-64",
          "timestamp": 1710590979402,
          "version": "8.6.0",
          "fn": "libcurl-8.6.0-hca28451_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libcurl-8.6.0-hca28451_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h0841786_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=12",
            "libzlib >=1.2.13,<1.3.0a0",
            "openssl >=3.1.1,<4.0a0"
          ],
          "license": "BSD-3-Clause",
          "license_family": "BSD",
          "md5": "1f5a58e686b13bcfde88b93f547d23fe",
          "name": "libssh2",
          "sha256": "50e47fd9c4f7bf841a11647ae7486f65220cfc988ec422a4475fe8d5a823824d",
          "size": 271133,
          "subdir": "linux-64",
          "timestamp": 1685837707056,
          "version": "1.11.0",
          "fn": "libssh2-1.11.0-h0841786_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libssh2-1.11.0-h0841786_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h659d440_0",
          "build_number": 0,
          "depends": [
            "keyutils >=1.6.1,<2.0a0",
            "libedit >=3.1.20191231,<3.2.0a0",
            "libedit >=3.1.20191231,<4.0a0",
            "libgcc-ng >=12",
            "libstdcxx-ng >=12",
            "openssl >=3.1.2,<4.0a0"
          ],
          "license": "MIT",
          "license_family": "MIT",
          "md5": "cd95826dbd331ed1be26bdf401432844",
          "name": "krb5",
          "sha256": "259bfaae731989b252b7d2228c1330ef91b641c9d68ff87dae02cbae682cb3e4",
          "size": 1371181,
          "subdir": "linux-64",
          "timestamp": 1692097755782,
          "version": "1.21.2",
          "fn": "krb5-1.21.2-h659d440_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/krb5-1.21.2-h659d440_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h166bdaf_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=10.3.0"
          ],
          "license": "LGPL-2.1-or-later",
          "md5": "30186d27e2c9fa62b45fb1476b7200e3",
          "name": "keyutils",
          "sha256": "150c05a6e538610ca7c43beb3a40d65c90537497a4f6a5f4d15ec0451b6f5ebb",
          "size": 117831,
          "subdir": "linux-64",
          "timestamp": 1646151697040,
          "version": "1.6.1",
          "fn": "keyutils-1.6.1-h166bdaf_0.tar.bz2",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/keyutils-1.6.1-h166bdaf_0.tar.bz2",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hd590300_5",
          "build_number": 5,
          "constrains": [
            "zlib 1.2.13 *_5"
          ],
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "Zlib",
          "license_family": "Other",
          "md5": "f36c115f1ee199da648e0597ec2047ad",
          "name": "libzlib",
          "sha256": "370c7c5893b737596fd6ca0d9190c9715d89d888b8c88537ae1ef168c25e82e4",
          "size": 61588,
          "subdir": "linux-64",
          "timestamp": 1686575217516,
          "version": "1.2.13",
          "fn": "libzlib-1.2.13-hd590300_5.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libzlib-1.2.13-hd590300_5.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h47da74e_1",
          "build_number": 1,
          "depends": [
            "c-ares >=1.23.0,<2.0a0",
            "libev >=4.33,<4.34.0a0",
            "libev >=4.33,<5.0a0",
            "libgcc-ng >=12",
            "libstdcxx-ng >=12",
            "libzlib >=1.2.13,<1.3.0a0",
            "openssl >=3.2.0,<4.0a0"
          ],
          "license": "MIT",
          "license_family": "MIT",
          "md5": "700ac6ea6d53d5510591c4344d5c989a",
          "name": "libnghttp2",
          "sha256": "1910c5306c6aa5bcbd623c3c930c440e9c77a5a019008e1487810e3c1d3716cb",
          "size": 631936,
          "subdir": "linux-64",
          "timestamp": 1702130036271,
          "version": "1.58.0",
          "fn": "libnghttp2-1.58.0-h47da74e_1.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libnghttp2-1.58.0-h47da74e_1.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hd590300_1",
          "build_number": 1,
          "constrains": [
            "pyopenssl >=22.1"
          ],
          "depends": [
            "ca-certificates",
            "libgcc-ng >=12"
          ],
          "license": "Apache-2.0",
          "license_family": "Apache",
          "md5": "9d731343cff6ee2e5a25c4a091bf8e2a",
          "name": "openssl",
          "sha256": "2c689444ed19a603be457284cf2115ee728a3fafb7527326e96054dee7cdc1a7",
          "size": 2865379,
          "subdir": "linux-64",
          "timestamp": 1710793235846,
          "version": "3.2.1",
          "fn": "openssl-3.2.1-hd590300_1.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/openssl-3.2.1-hd590300_1.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hd590300_2",
          "build_number": 2,
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "BSD-2-Clause",
          "license_family": "BSD",
          "md5": "172bf1cd1ff8629f2b1179945ed45055",
          "name": "libev",
          "sha256": "1cd6048169fa0395af74ed5d8f1716e22c19a81a8a36f934c110ca3ad4dd27b4",
          "size": 112766,
          "subdir": "linux-64",
          "timestamp": 1702146165126,
          "version": "4.33",
          "fn": "libev-4.33-hd590300_2.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libev-4.33-hd590300_2.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "he28a2e2_2",
          "build_number": 2,
          "depends": [
            "libgcc-ng >=7.5.0",
            "ncurses >=6.2,<7.0.0a0"
          ],
          "license": "BSD-2-Clause",
          "license_family": "BSD",
          "md5": "4d331e44109e3f0e19b4cb8f9b82f3e1",
          "name": "libedit",
          "sha256": "a57d37c236d8f7c886e01656f4949d9dcca131d2a0728609c6f7fa338b65f1cf",
          "size": 123878,
          "subdir": "linux-64",
          "timestamp": 1597616541093,
          "version": "3.1.20191231",
          "fn": "libedit-3.1.20191231-he28a2e2_2.tar.bz2",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libedit-3.1.20191231-he28a2e2_2.tar.bz2",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hd590300_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "MIT",
          "license_family": "MIT",
          "md5": "f6afff0e9ee08d2f1b897881a4f38cdb",
          "name": "c-ares",
          "sha256": "2a5866b19d28cb963fab291a62ff1c884291b9d6f59de14643e52f103e255749",
          "size": 163578,
          "subdir": "linux-64",
          "timestamp": 1708684786032,
          "version": "1.27.0",
          "fn": "c-ares-1.27.0-hd590300_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/c-ares-1.27.0-hd590300_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hd590300_5",
          "build_number": 5,
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "bzip2-1.0.6",
          "license_family": "BSD",
          "md5": "69b8b6202a07720f448be700e300ccf4",
          "name": "bzip2",
          "sha256": "242c0c324507ee172c0e0dd2045814e746bb303d1eb78870d182ceb0abc726a8",
          "size": 254228,
          "subdir": "linux-64",
          "timestamp": 1699279927352,
          "version": "1.0.8",
          "fn": "bzip2-1.0.8-hd590300_5.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/bzip2-1.0.8-hd590300_5.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hbcca054_0",
          "build_number": 0,
          "depends": [],
          "license": "ISC",
          "md5": "2f4327a1cbe7f022401b236e915a5fef",
          "name": "ca-certificates",
          "sha256": "91d81bfecdbb142c15066df70cc952590ae8991670198f92c66b62019b251aeb",
          "size": 155432,
          "subdir": "linux-64",
          "timestamp": 1706843687645,
          "version": "2024.2.2",
          "fn": "ca-certificates-2024.2.2-hbcca054_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/ca-certificates-2024.2.2-hbcca054_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        }
      ],
      "run_exports": {
        "gcc_linux-64": {
          "strong": [
            "libgcc-ng >=13"
          ]
        }
      }
    },
    "host": {
      "specs": [
        {
          "source": "raw",
          "spec": "zlib"
        },
        {
          "source": "run_export",
          "spec": "libgcc-ng >=13",
          "from": "build",
          "source_package": "gcc_linux-64"
        }
      ],
      "resolved": [
        {
          "build": "h807b86a_5",
          "build_number": 5,
          "constrains": [
            "libgomp 13.2.0 h807b86a_5"
          ],
          "depends": [
            "_libgcc_mutex 0.1 conda_forge",
            "_openmp_mutex >=4.5"
          ],
          "license": "GPL-3.0-only WITH GCC-exception-3.1",
          "license_family": "GPL",
          "md5": "d4ff227c46917d3b4565302a2bbb276b",
          "name": "libgcc-ng",
          "sha256": "d32f78bfaac282cfe5205f46d558704ad737b8dbf71f9227788a5ca80facaba4",
          "size": 770506,
          "subdir": "linux-64",
          "timestamp": 1706819192021,
          "version": "13.2.0",
          "fn": "libgcc-ng-13.2.0-h807b86a_5.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libgcc-ng-13.2.0-h807b86a_5.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "conda_forge",
          "build_number": 0,
          "depends": [],
          "license": "None",
          "md5": "d7c89558ba9fa0495403155b64376d81",
          "name": "_libgcc_mutex",
          "sha256": "fe51de6107f9edc7aa4f786a70f4a883943bc9d39b3bb7307c04c41410990726",
          "size": 2562,
          "subdir": "linux-64",
          "timestamp": 1578324546067,
          "version": "0.1",
          "fn": "_libgcc_mutex-0.1-conda_forge.tar.bz2",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/_libgcc_mutex-0.1-conda_forge.tar.bz2",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "2_gnu",
          "build_number": 16,
          "constrains": [
            "openmp_impl 9999"
          ],
          "depends": [
            "_libgcc_mutex 0.1 conda_forge",
            "libgomp >=7.5.0"
          ],
          "license": "BSD-3-Clause",
          "license_family": "BSD",
          "md5": "73aaf86a425cc6e73fcf236a5a46396d",
          "name": "_openmp_mutex",
          "sha256": "fbe2c5e56a653bebb982eda4876a9178aedfc2b545f25d0ce9c4c0b508253d22",
          "size": 23621,
          "subdir": "linux-64",
          "timestamp": 1650670423406,
          "version": "4.5",
          "fn": "_openmp_mutex-4.5-2_gnu.tar.bz2",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/_openmp_mutex-4.5-2_gnu.tar.bz2",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "h807b86a_5",
          "build_number": 5,
          "depends": [
            "_libgcc_mutex 0.1 conda_forge"
          ],
          "license": "GPL-3.0-only WITH GCC-exception-3.1",
          "license_family": "GPL",
          "md5": "d211c42b9ce49aee3734fdc828731689",
          "name": "libgomp",
          "sha256": "0d3d4b1b0134283ea02d58e8eb5accf3655464cf7159abf098cc694002f8d34e",
          "size": 419751,
          "subdir": "linux-64",
          "timestamp": 1706819107383,
          "version": "13.2.0",
          "fn": "libgomp-13.2.0-h807b86a_5.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libgomp-13.2.0-h807b86a_5.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hd590300_0",
          "build_number": 0,
          "depends": [
            "libgcc-ng >=12",
            "libzlib 1.3.1 hd590300_0"
          ],
          "license": "Zlib",
          "license_family": "Other",
          "md5": "1a3d399cae6848faf424a85065c41f52",
          "name": "zlib",
          "sha256": "516dc1ec7617be54a39c11213385e13c67f6edfbaa12a628280a13c746a31ae9",
          "size": 93000,
          "subdir": "linux-64",
          "timestamp": 1709240893729,
          "version": "1.3.1",
          "fn": "zlib-1.3.1-hd590300_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/zlib-1.3.1-hd590300_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        },
        {
          "build": "hd590300_0",
          "build_number": 0,
          "constrains": [
            "zlib 1.3.1 *_0"
          ],
          "depends": [
            "libgcc-ng >=12"
          ],
          "license": "Zlib",
          "license_family": "Other",
          "md5": "ce6e041b5727209dc79fc05f260839bc",
          "name": "libzlib",
          "sha256": "9df76697b6fe2228711283c0dda589354d5df633726db2fca7f7841dd5e50c82",
          "size": 61797,
          "subdir": "linux-64",
          "timestamp": 1709240883161,
          "version": "1.3.1",
          "fn": "libzlib-1.3.1-hd590300_0.conda",
          "url": "https://conda.anaconda.org/conda-forge/linux-64/libzlib-1.3.1-hd590300_0.conda",
          "channel": "https://conda.anaconda.org/conda-forge/"
        }
      ],
      "run_exports": {
        "zlib": {
          "weak": [
            "libzlib >=1.3.1,<1.4.0a0"
          ]
        }
      }
    },
    "run": {
      "depends": [
        {
          "source": "run_export",
          "spec": "libzlib >=1.3.1,<1.4.0a0",
          "from": "host",
          "source_package": "zlib"
        },
        {
          "source": "run_export",
          "spec": "libgcc-ng >=13",
          "from": "build",
          "source_package": "gcc_linux-64"
        }
      ],
      "constrains": [],
      "run_exports": null
    }
  },
  {
    "build": null,
    "host": null,
    "run": {
      "depends": [],
      "constrains": [],
      "run_exports": null
    }
  }
]

```

</details>

Isn't that the desired output?

Reply:
Ah! render_output: Vec<FinalizedDependencies> is already a Vector!

Reply:
Yes - thanks for the suggestion